README.md

```diff
@@ -1 +1,47 @@
-# Hello-Microverse
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# "Hello Microverse" project
+
+> Create branches with different base branches.
+Understand the concept of version control.
+Follow GitHub flow.
+Open pull requests in GitHub.
+Write descriptive, easy to understand commit messages.
+Write short, detailed, easy to understand descriptions of pull requests.
+Write a descriptive and easy to understand README in business English for every project submitted.
+Demonstrate an ability to submit a project for code review.
+
+## Built With
+
+- HTML
+- CSS
+- github workflow
+
+### Prerequisites
+-HTML
+-CSS
+
+## Authors
+
+👤 **Author1**
+
+- GitHub: [@github/telanet/Hello-Microverse](https://github.com/githubhandle)
```

Comment:
- [ ] Perfect!, you documentation looks professional. Let's just make sure we use `[@our_github_username]` instead of github/talent/Hello-Microverse, this should also be a link to your GitHub profile.
Should look something like this 
```MD
[@telanet](https://github.com/telanet)
```

tip: if you are using VScode as your code editor and want to preview your readme file you could use CTRL + SHIFT + V